src/platform/user/profile/vet360/components/base/Vet360ProfileFieldHeading.jsx

```diff
@@ -11,6 +11,7 @@ function Vet360ProfileFieldHeading({ children, onEditClick }) {
           type="button"
           data-action="edit"
           onClick={onEditClick}
+          id={fieldName}
```

Comment:
Maybe make this `id` a little more specific to reduce the likelihood of accidentally having identical `id`s on the same page (very unlikely, I know, but might as well nip it in the bud now)